init-tools.sh

```diff
@@ -99,14 +118,24 @@ if [ ! -e $__DOTNET_PATH ]; then
     echo "Installing dotnet cli..."
     __DOTNET_LOCATION="https://dotnetcli.azureedge.net/dotnet/Sdk/${__DOTNET_TOOLS_VERSION}/${__DOTNET_PKG}.tar.gz"
     # curl has HTTPS CA trust-issues less often than wget, so lets try that first.
```

Comment:
This comment should be moved, too. (And IMO it should really be just above the `if command -v curl > /dev/null; then`, since that's what it's describing.)

Reply:
Fixed.